ecoindex_cli/cli/app.py

```diff
@@ -102,14 +112,27 @@ def analyze(
         length=len(urls) * len(window_sizes),
         label="Processing",
     ) as progress:
+        err_found = False
```

Comment:
```suggestion
        error_found = False
```

You can be explicit :)